Documentation/Signoffs/Microcontroller.md

```diff
@@ -0,0 +1,55 @@
+# Microcontroller Signoff
+## FUNCTION OF THE SUBSYSTEM
+The microcontroller and its code are the centerpieces of the entire device.  They are responsible for operating, managing, communicating with, maintaining, and collaborating with all of the other subsystems and acts as both the brain and body of the device.  All the data and instructions must flow through the microcontroller at some point in its lifetime.  This gives a common place for all of the data to go and be readily available for whichever subsystem has need of it.  The hardware will be responsible for the actual actions and completion of any given task.  The code will be responsible for controlling the hardware and manipulating data.  This subsystem will be where most of the coding for the project is done as the operating code will be the largest by far.  The code itself will be where the “magic” happens as it allows for interfacing with the hardware and makes it more simple to utilize.
+
+These jobs require a sufficiently powerful board.  One that has ample storage, plenty of physical space, decent processing power, sufficient support, and none restrictive.  Before choosing components and detailing the design, the specific constraints of the subsystem need to be known and laid out.  Then any potential solutions can be compared to the constraints to test viability and fulfillment of the subsystem’s purpose. 
+
+## CONSTRAINTS ON THE SUBSYSTEM
+The subsystem constraints are placed to ensure that it can fulfill its job in a satisfactory manner.  The constraints given below are generalized to allow for comparison of any potential contender for the microcontroller.  Constraints on the subsystem are as follows:
+1. The main job of the microcontroller will be to manage and operate the other subsystems to work cooperatively and synchronously.  The board to be chosen must support fairly sophisticated code and be powerful enough to head all of the other subsystems.
+2. The microcontroller board should not be a hefty price for the user budget.  With a budget of $300, a super-powered board is not affordable.  The board price ranges between $20-$60.  This range is gathered from estimating the costs of other components and reasoning where money can be saved.
+3. The microcontroller must support communication between components.  To do this, Serial Peripheral Interface (SPI), Universal Asynchronous Receiver/Transmitter (UART), or some form of hardware communication will need to be achievable on the board.  SPI is a synchronous method while UART is an asynchronous method for two devices connected over a short distance to liaise.
```

Comment:
It seems that this and #5 are covering the same thing. Combine the two. (If this is related to #4 it should be deleted - in that case it would be an internal subsystem constraint, not an external constraint, because some microcontrollers directly support USB peripherals while others need extra modules.)